Phonebook.Frontend/src/styles/overrides.scss

```diff
@@ -22,6 +22,19 @@ $config: mat-typography-config();
   height: inherit;
 }
 
+// This fixes https://github.com/angular/components/issues/11094
+mat-card.mat-card{ //two times mat-card to override component styling of the material component (through higher specificity)
+  display: flex;
+  flex-direction: column;
+  flex: 1;
```

Comment:
Seems like the flex: 1 breaks other format, so remove it here and add `pb-flex-1` to each card.

Reply:
@DanielHabenicht this is the only thing which should done before merge